### DISCUSSION TITLE
[Wix4] Generation of file list (formerly heat.exe)

### DISCUSSION THREAD

**Discussion by dd-mozdemir** [wixtoolset/issues#6746] [Questions] [answered]
Hi there,

I could not find a lot of documentation regarding WixToolset v4 and I've been working through converting a project to see what works and what doesn't. 

I was wondering what the new method of generating file lists is with wixtoolset v4? I know that `heat.exe` was used previously but I'm not sure how to generate this now.  Previously the command would look like this:
 ```
heat.exe dir C:\foo\bar -nologo -srd -sreg -gg -cg ProjectDir -dr PROJECTROOT -var var.ProjectSourceDir -out projectfiles.wxs
```

Thanks!

**Comment by rseanhall** [comment, marked as answer]
Heat was not part of the planned features in preview.0, so how to do that in v4 hasn't been decided yet. But if you really want to play around with what the state of Heat was at that time, you can find `heat.exe` in the contents of the WixToolset.Sdk.nupkg in the artifacts.zip in the Github release. MSBuild support was also not part of the planned features in preview.0 so you shouldn't expect anything to work from that .nupkg.
